data/remnant/remnant 2 side missions.txt

```diff
@@ -144,3 +144,224 @@ mission "Remnant: Visit to Paradise 1"
+	invisible
+	source
+		government "Remnant"
+	destination "Viminal"
+	to offer
+		has "event: remnant: cognizance calm"
+		# random < 30
+	npc "scan outfits"
+		government "Ember Waste"
+		personality uninterested waiting staying mute
+		system Caeculus
+		ship "Dromedary Ghost" "???"
+		dialog "This ship barely registers on your sensors, and you can clearly see through it. The Remnant will want to know about it."
+	on complete
+		conversation
+			`Looking around the Viminal spaceport, you manage to spot Taely using something that looks kind of like a small catapult to throw a large rock across the cavern. A nearby Merganser leaps into the air and chases after it. The ship pounces on the rock, vanishes from sight, and then reappears next to Taely and drops the rock next to her. Without looking away from the Merganser she waves you over.`
+			`	"Well met, <first>, how goes your travels, found anything interesting?`
+			`	She listens to your description of the encounter with the ghostly ship, and looks at the recording you pull up from your commlink. "This is very intriguing. I will need to consult a few people. If you are interested in investigating this, ensure your ship has a research laboratory installed, and meet me in the starport."`
+
+
+
+mission "Remnant: Shattered Light 2"
+	name "Searching for Ghosts"
+	description "Take Ruach to find and scan the ghostly ship."
+	blocked "Your commlink chimes with a note that this mission requires a Research Laboratory as well as 2 cargo space and 1 free bunk."
+	cargo "equipment" 2
+	passengers 1
+	source "Viminal"
+	to offer
+		has "Remnant: Shattered Light 1: done"
+	npc "scan outfits"
+		government "Ember Waste"
+		personality uninterested waiting staying mute
+		system Levana
+		ship "Dromedary Specter" "???"
+		dialog "This ship seems a bit more solid than it did when you first saw it, but it does not seem to be quite present."
+	on offer
+		require "Research Laboratory"
+		conversation
+			`Your arrival in the starport finds Taely in deep conversation with an elderly Remnant, empty food trays cluttering the table before them. They turn as one when you approach, breaking off their conversation.`
+			`	"Thank you for coming, Captain <last>. I would like to introduce you to Ruach. She was a shipyard engineering prefect for many decades, but now spends most of her time as a teacher." Ruach gives a polite bow. "Taely lays too many compliments on me," she gestures with amusement. "I fulfilled my duty doing the best with that which the Embers blessed me, nothing more, nothing less. But it has been rewarding watching someone take everything I taught them and fly with it to heights I could only dream of."`
+			`	Ruach's gestures take on a more precise cadence. "Taely tells me that you have spotted a strange 'ghost' in Caeculus that bears a resemblance to ancestral ships. Could you take me to do a scan of it please?`
+			choice
+				`	"OK"`
+					accept
+				`	"Sorry, not interested."`
+					decline
```

Comment:
I guess here you don't get many options because she is in a hurry to see the ship? Maybe mention that, I wish I could ask her about remnant education here ^^
Also did you forget a closing "? (just before choice)

Reply:
That is correct: She's in a hurry, so not too much conversation here. But I will keep that in mind as a topic to include. I do intend to do follow-up missions to the Shattered Light set, so I'll remember to include a bit on education at that time.

And yes, I did miss a closing quotation mark there. Thanks!